### PR TITLE
Dread: Some updates to make connections no longer "dangerous", or only "dangerous" when using certain settings

### PR DESCRIPTION
**Pull request by Tyranisaur**

### CODE REVIEW

**Review by henriquegemignani — APPROVED** [review]
Please resolve the conflicts

**Comment by henriquegemignani** [comment]
Oh and add a better PR name please

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,10 +27,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Changed: in Burenia - Main Hub Tower Middle, lowering the Spider Magnet Wall is now "dangerous" only when Highly Dangerous Logic is enabled. The connection from the bottom of the room to the Pickup Platform that uses Grapple Movement requires the Spider Magnet Wall to not be lowered now requires Highly Dangerous Logic. The randomizer currently doesn't have the necessary options to make this connection mandatory in any seeds anyway.
+- Changed: In Dairon - Teleport to Artaria, breaking the speed blocks is no longer "dangerous". This is done by removing the "Before Event" condition on breaking the blocks from above.
+- Changed: In Artaria - Water Reservoir, breaking the blob is no longer "dangerous", as long as Slide is not randomized. This was previously dangerous because there's a connection in EMMI Zone Exit Southwest that makes use of Speed Booster, however, by simply adding a "Can Slide" option on the same condition, the logic now sees the blob as safe.
```
Comment (ThanatosGit) [inline CHANGELOG.md:36]: duplicate

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,10 +27,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Changed: In Burenia: Fighting Drogyga is now only "dangerous" if Highly Dangerous Logic is enabled. This is achieved by adding a Highly Dangerous Logic constraint on all instances where the logic uses "Before Drogyga" on connections in the Underneath Drogyga room.
```
Comment (ThanatosGit) [inline CHANGELOG.md:37]: same

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,10 +27,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Changed: In Burenia - Main Hub Tower Middle, lowering the Spider Magnet Wall is now "dangerous" only when Highly Dangerous Logic is enabled. The connection from the bottom of the room to the Pickup Platform that uses Grapple Movement requires the Spider Magnet Wall to not be lowered now requires Highly Dangerous Logic. The randomizer currently doesn't have the necessary options to make this connection mandatory in any seeds anyway.
```
Comment (ThanatosGit) [inline CHANGELOG.md:38]: same

**Review by ThanatosGit — CHANGES_REQUESTED** [review]
Sorry, something went wrong when resolving merge conflicts.

File: randovania/games/dread/json_data/Dairon.txt [hunk 4]
```diff
@@ -1417,6 +1417,8 @@ Extra - asset_id: collision_camera_018
 > Dock to EMMI Zone Exit North; Heals? False
   * Layers: default
   * Open Passage to EMMI Zone Exit North/Dock to Yellow EMMI Introduction
+  > Dock to Big Hub
+      Speed Booster and After Dairon - Yellow Introduction Speed Blocks Destroyed
```
Comment (XenoWars) [inline randovania/games/dread/json_data/Dairon.txt:1421]: This connection is redundant. It's already handled from "Door to EMMI Zone Exit East"

**Review by XenoWars — CHANGES_REQUESTED** [review]

File: randovania/games/dread/json_data/Dairon.txt (hunk 4 above)
Comment (Tyranisaur) [reply randovania/games/dread/json_data/Dairon.txt]: I missed that, the point was to do it trickless. Given that I found other cases that indicate that Speed Booster can be used tricklessly to pass Shutter Gates, I changed the existing connection by moving the Speed Booster branch outside the Flash Shift Skip branch.

File: CHANGELOG.md (hunk 1 above)
Comment (Tyranisaur) [reply CHANGELOG.md:31]: Please forgive me for using a rebase to resolve the bad merge in the merge commit. That seemed like the easiest option.

Comment (ThanatosGit) [reply CHANGELOG.md:31]: Haha, no worries. Git can be an ass sometimes with how it is showing changes / merge conflicts

**Comment by XenoWars** [comment]
Copied from discord: if anything the burenia one should force speed conservation at least, not just from the suction enemy. And if it does, the suction enemy one probably wouldnt be needed either.

but yeah maybe make the cataris and dairon ones trickless. ferenia can be moved to speed conservation perhaps.

**Comment by ThanatosGit** [comment]
@XenoWars Can you approve that? I'm asking because you asked for the recent changes.

File: randovania/games/dread/json_data/Burenia.txt [hunk 5]
```diff
@@ -749,7 +749,7 @@ Extra - asset_id: collision_camera_008
               Flash Shift or Space Jump
               All of the following:
                   # Carry it from the Map Station, or alternatively from the suction enemy below
-                  Speed Booster
+                  Speed Booster and Speed Booster Conservation (Beginner)
                   Any of the following:
                       Disabled Door Lock Randomizer
                       After Burenia - Ferenia Teleport Grapple Block and Knowledge (Beginner) and Speed Booster Conservation (Intermediate)
```
Comment (XenoWars) [inline randovania/games/dread/json_data/Burenia.txt:755]: This one is redundant now due to speed conservation being added above.

File: randovania/games/dread/json_data/Cataris.txt [hunk 6]
```diff
@@ -237,7 +237,10 @@ Extra - asset_id: collision_camera_004
 > Front of Thermal Door; Heals? False
   * Layers: default
   > Dock to Total Recharge Station South
-      Before Cataris - First Thermal Device
+      Any of the following:
+          Before Cataris - Final Thermal Device
```
Comment (XenoWars) [inline randovania/games/dread/json_data/Cataris.txt:241]: Whoops? This should be First, not Final.

File: randovania/games/dread/json_data/Cataris.txt [hunk 7]
```diff
@@ -361,7 +364,10 @@ Extra - asset_id: collision_camera_007
   > Door to Thermal Device Room South (Thermal Device)
       Trivial
   > Dock to Thermal Device Room South
-      Before Cataris - First Thermal Device
+      Any of the following:
+          Before Cataris - Final Thermal Device
```
Comment (XenoWars) [inline randovania/games/dread/json_data/Cataris.txt:368]: Should be First Thermal Device.

**Review by XenoWars — CHANGES_REQUESTED** [review]

File: randovania/games/dread/json_data/Cataris.txt [hunk 8]
```diff
@@ -361,7 +364,10 @@ Extra - asset_id: collision_camera_007
   > Door to Thermal Device Room South (Thermal Device)
       Trivial
   > Dock to Thermal Device Room South
-      Before Cataris - First Thermal Device
+      Any of the following:
+          Before Cataris - First Device Blob
```
Comment (XenoWars) [inline randovania/games/dread/json_data/Cataris.txt:368]: Still wrong.

**Review by XenoWars — CHANGES_REQUESTED** [review]

**Review by XenoWars — APPROVED** [review]

**Pull request merged**